e2e/custom.ini

```diff
@@ -0,0 +1,5 @@
+[live_performance]
+measure_data_render_delay = true
```

Comment:
Is this required?  Do we need a special config section, or can/should we interact with it with it at runtime?   If we require a special setup, then we can only measure instances that have explicit configs in their setup.

Reply:
I stopped using it already after the last discussion but didn't remove the config - its gone now